src/application.rs

```diff
@@ -168,6 +197,13 @@ impl Application {
         };
         self.user_interface.key_pressed(keycode);
     }
+
+    fn selected_face(&self) -> Option<UniqueId> {
+        let Ok([&face_id]) = self.selected_faces.iter().next_chunk() else {
+            return None
+        };
+        Some(face_id)
```

Comment:
```suggestion
        self.selected_faces.iter().next().map(|&id| id)
```

Reply:
What if there are two?  You just get the first, right?  I don't want any if there are two.

Reply:
Then neither solution works. next_chunk() will still give you the next one element, whether or not there are more.